lib/mnit/tileset.nit

```diff
@@ -84,6 +84,30 @@ class TileSetFont
 		if i == -1 then return null
 		return subimages[i]
 	end
+
+	# Distance between the beginning of a letter and the beginning of the next letter
```

Comment:
"of a letter **tile**  and the beginning of the next letter **tile**" My first reaction what that this included some kind of kerning, so I said "nice", then I read the code and was disappointed.